features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesStateProvider.kt

```diff
@@ -68,6 +68,7 @@ fun aMessagesState() = MessagesState(
     customReactionState = CustomReactionState(
         selectedEventId = null,
         eventSink = {},
+        selectedEmoji = setOf(),
```

Comment:
```suggestion
        selectedEmoji = persistentSetOf(),
```

This quickly instantiates a `PersistentSet`, which is an implementation of `ImmutableSet`. You'll need to add imports for it too.